### ISSUE TITLE
Training on Funsd dataset gives me only 0.68 F1 score on the test set

### ISSUE THREAD

**Issue by SwathiYanture** [microsoft/unilm#277]
**Describe**
Model I am using (LayoutLM ...):I have followed the steps mentioned in the readme file on Funsd dataset. The F1 score reported in the paper 0.7866.
Below are the parameters I am using:
pretrained model - LayoutLM-Large, Uncased (11M documents, 2 epochs): 24-layer, 1024-hidden, 16-heads, 343M parameters 
max_seq_length - 512
num_train_epochs - 100.0
per_gpu_train_batch_size - 12
per_gpu_eval_batch_size - 16 
rest of the parameters are the default parameters that are present in the code

I have run unilm/layoutlm/examples/seq_labeling/run_seq_labeling.py without making any changes to the code. Is there something I am missing here?
 
Thanks.

**Comment by clemsage** [comment]
The experiments reported in the paper were done with a batch size of 16, so a batch size of 12 might slightly change the test F1 score but a gap of 0.1 seems big. I have also tried to reproduce the results without making any change to the code, I reach a F1 score of 0.7841 for the base model compared to 0.866 in the paper !

**Comment by wolfshow** [comment]
@clemsage Do you mean 0.7866 in the paper?

**Comment by clemsage** [comment]
@wolfshow Yes, sorry for the typo, I've fixed it !

**Comment by wolfshow** [comment]
@SwathiYanture Can you please share the command you are running?

**Comment by SwathiYanture** [comment]
I had set the default values in the code and executed, probably one of the parameter's value was missed.
I got **f1 = 0.7796202835856765** for the base model with the below command
 python run_seq_labeling.py  --data_dir data --model_type layoutlm --model_name_or_path pretrained_models/layoutlm-base-uncased --do_lower_case --max_seq_length 512 --do_train --num_train_epochs 100.0 --logging_steps 10 --save_steps -1 --output_dir model-output-2 --labels data/labels.txt --per_gpu_train_batch_size 12 --per_gpu_eval_batch_size 12

Thank you for the response @clemsage @wolfshow

**Comment by tengerye** [comment]
@SwathiYanture Hi, may I ask how did you run do_eval please? I ran `--do_train --do_eval` and got 0.176 F1-score tragically.

**Comment by SwathiYanture** [comment]
> @SwathiYanture Hi, may I ask how did you run do_eval please? I ran `--do_train --do_eval` and got 0.176 F1-score tragically. 

I used the exact run command written above. Which dataset are you using? Is it the funsd dataset or your own application specific dataset?

**Comment by tengerye** [comment]
> > @SwathiYanture Hi, may I ask how did you run do_eval please? I ran `--do_train --do_eval` and got 0.176 F1-score tragically.
> 
> I used the exact run command written above. Which dataset are you using? Is it the funsd dataset or your own application specific dataset?

It is the FUNSD dataset. Did you download the pre-trained model yourself from the links provided by this repo?
